test-worker/model/Order/index.ts

```diff
@@ -0,0 +1,19 @@
+export interface Order {
+	id: string
+	number: string
+	amount: number
+	currency: string
+}
+
+export namespace Order {
+	export function is(value: any | Order): value is Order {
+		return (
+			value &&
+			typeof value == "object" &&
+			typeof value.id == "string" &&
+			typeof value.number == "number" &&
```

Comment:
typeof value.number == "number" should be "string" on line 14 or the typing of property number should be changed.